backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/llvm/IrToBitcode.kt

```diff
@@ -713,8 +711,8 @@ internal class CodeGeneratorVisitor(val context: Context, val lifetimes: Map<IrE
                 declaration.location(start = false)) {
             using(FunctionScope(declaration, it)) {
                 val parameterScope = ParameterScope(declaration, functionGenerationContext)
-                using(parameterScope) {
-                    using(VariableScope()) {
+                using(parameterScope) usingParameterScope@ {
+                    using(VariableScope()) usingVariableScope@ {
```

Comment:
Is this formatted according to [Kotlin coding conventions](https://kotlinlang.org/docs/reference/coding-conventions.html)?

Reply:
Also please recheck `return@using` below.